core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerProcess.java

```diff
@@ -110,7 +110,7 @@
       // registered at worker registry, so the maximum timeout here is set to the multiply of
       // the number of factories by the default timeout of getting a worker from the registry.
       CommonUtils.invokeAll(callables,
-          (long) callables.size() * 10 * Constants.DEFAULT_REGISTRY_GET_TIMEOUT_MS);
```

Comment:
do we still need the definition of this constant?  is it a good time to remove it also?

Reply:
I found this param is used in other area, so I didn't remove it.